english/makesmoores/makesmoores

```diff
@@ -0,0 +1,3 @@
+You build a camp-fire, grab a fistful of the wall and put it on a stick.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.